rust/ql/lib/codeql/rust/frameworks/rustcrypto/rustcrypto.model.yml

```diff
@@ -1,10 +1,10 @@
 extensions:
   - addsTo:
       pack: codeql/rust-all
-      extensible: sinkModelDeprecated
+      extensible: sinkModel
     data:
-      - ["repo:https://github.com/RustCrypto/traits:digest", "<_ as crate::digest::Digest>::new_with_prefix", "Argument[0]", "hasher-input", "manual"]
-      - ["repo:https://github.com/RustCrypto/traits:digest", "<_ as crate::digest::Digest>::update", "Argument[0]", "hasher-input", "manual"]
-      - ["repo:https://github.com/RustCrypto/traits:digest", "<_ as crate::digest::Digest>::chain_update", "Argument[0]", "hasher-input", "manual"]
-      - ["repo:https://github.com/RustCrypto/traits:digest", "<_ as crate::digest::Digest>::digest", "Argument[0]", "hasher-input", "manual"]
-      - ["repo:https://github.com/stainless-steel/md5:md5", "crate::compute", "Argument[0]", "hasher-input", "manual"]
+      - ["<_ as digest::digest::Digest>::new_with_prefix::new_with_prefix", "Argument[0]", "hasher-input", "manual"]
+      - ["<_ as digest::digest::Digest>::new_with_prefix::update", "Argument[0]", "hasher-input", "manual"]
+      - ["<_ as digest::digest::Digest>::new_with_prefix::chain_update", "Argument[0]", "hasher-input", "manual"]
+      - ["<_ as digest::digest::Digest>::new_with_prefix::digest", "Argument[0]", "hasher-input", "manual"]
```

Comment:
```suggestion
      - ["<_ as digest::digest::Digest>::new_with_prefix", "Argument[0]", "hasher-input", "manual"]
      - ["<_ as digest::digest::Digest>::update", "Argument[0]", "hasher-input", "manual"]
      - ["<_ as digest::digest::Digest>::chain_update", "Argument[0]", "hasher-input", "manual"]
      - ["<_ as digest::digest::Digest>::digest", "Argument[0]", "hasher-input", "manual"]
```

Reply:
Yep, it looks like I botched some edits.